aat/src/resources/features/F-023/S-005.td.json

```diff
@@ -0,0 +1,32 @@
+{
+  "title": "Must return a negative response on non existent Case ID",
+  "_guid_": "S-005",
+  "_extends_": "F-023_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD - wants to see a list of valid documents they can print",
+    "contains a non existent caseID"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "cid": "1111222233334444"
+    }
+    },
+
+  "expectedResponse": {
+    "responseCode": 404,
+    "responseMessage": "Not Found",
```

Comment:
Please extent the common 404 response object. That will bring the 2 lines here and the headers below.

Reply:
done